app/assets/javascripts/welcome.js

```diff
@@ -0,0 +1,13 @@
+// Loads all Bootstrap javascripts
+//= require jquery
+//= require jquery_ujs
+//= require bootstrap
+//= require footer_fixed.js
+//= require browser_timezone_rails/application.js
+
+$("a.github_auth").click(function () {
```

Comment:
__[eslint]__
*  Error - "$" is not defined. (no-undef)

*  Error - Missing "use strict" statement. (strict)